### ISSUE TITLE
Mission Complete Screens

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1323]
I was thinking that the mission complete screens should show not just the route information on the top-down map (previous routes walked in neighborhood + current route walked as part of mission) but also:

- All previous labels contributed for that neighborhood by that user
- All current labels contributed for that neighborhood by that user for the current mission

To distinguish these two things, maybe previous labels are all gray?

**Comment by misaugstad** [comment]
Yep I really like this idea, and shouldn't be terribly hard to implement, either!

**Comment by misaugstad** [comment]
Still valid and now smaller than it was. The mission-complete map was rewritten (`ModalMissionCompleteMap.js`) and already draws three street tiers — this mission, your earlier missions, and the community's — but labels come from a single layer of just the labels placed during this mission.

So the remaining work is one more label layer: the user's earlier labels in the neighborhood, styled muted/gray behind the fresh ones. The rendering path (`addLabelsToMap`) and the this-mission/previous split are both already there; it mainly needs the endpoint to return the earlier labels too.

Related: #4820 makes the same peak–end argument for the tutorial's ending screen.
